app/components/Footer/index.js

```diff
@@ -2,34 +2,89 @@ import React from 'react';
 import { FormattedMessage } from 'react-intl';
 
 import A from 'components/A';
-import LocaleToggle from 'containers/LocaleToggle';
 import styled from 'styled-components';
+import NormalImg from 'components/Img';
+import { Link } from 'react-router-dom';
 import messages from './messages';
+import Logo from './logo.png';
+import Fcb from './fcb.svg';
 
 const Wrapper = styled.footer`
   display: flex;
   justify-content: space-between;
-  padding: 3em 0;
+  align-items: center;
+  padding: 2em;
   border-top: 1px solid #666;
+  position: relative;
+
+  @media (min-width: 768px) {
+    padding: 2em 4em;
+  }
+`;
+
+const FcbLink = styled(Link)`
```

Comment:
what does it mean fcb ?

Reply:
Facebook Link. This is in footer and it'a a link with url to facebook group. It isn't ok?